src/components/Positions.jsx

```diff
@@ -545,7 +488,6 @@ const PositionContainer = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
-  margin-top: 10px;
   /* flex: 1 1 0px; */
   width: auto;
```

Comment:
I like all the code cleanup!  I think it might be a good idea to add some code here to reduce the width on mobile here. It looks like this currently:
![IMG_6929](https://user-images.githubusercontent.com/91639023/215829054-7475e408-aec6-4568-aadf-065574f3a1f0.PNG)

It looks ok, but some of the container is spilling outside the screen. I think it looks better like this:
![IMG_6927](https://user-images.githubusercontent.com/91639023/215829542-89b670ea-7c01-403e-b38f-829698675be3.PNG)

I found the following code to work (need to add mobile prop in: `<PositionsContainer mobile={mobile}>`) and add the following code.

```
${(props) => props.mobile && css`
    width: 90%;
    margin: auto;
  `}
```

Reply:
I think you should wrap the 2 lines you added with a mobile conditional (like the one I left in the comment above) because the max-width is affecting desktop as well. It looks like the following:
<img width="879" alt="Screen Shot 2023-02-01 at 5 56 38 PM" src="https://user-images.githubusercontent.com/91639023/216184738-c02e1484-35b6-4206-87f5-309555f74a7d.png">

Reply:
It looks good on mobile though.